tests/test_other.py

```diff
@@ -8290,7 +8290,7 @@ def strip_numeric_suffixes(funcname):
     'O3': (['-O3'], [], [],          85), # noqa
     'Os': (['-Os'], [], [],          85), # noqa
     'Oz': (['-Oz'], [], [],          85), # noqa
-    'Os_mr': (['-Os', '-s', 'MINIMAL_RUNTIME'], [], [], 85), # noqa
+    'Os_mr': (['-Os', '-s', 'MINIMAL_RUNTIME', '--no-entry'], [], [], 85), # noqa
```

Comment:
this is not standalone and so it seems unrelated?

Reply:
Oops, good point.  Not sure how that happened.